R/mod_scrollytell_example.R

```diff
@@ -4,89 +4,104 @@
 #'
 #' @param id,input,output,session Internal parameters for {shiny}.
 #'
-#' @noRd 
-#'
-#' @importFrom shiny NS tagList 
+#' @noRd
 mod_scrollytell_example_ui <- function(id){
   ns <- NS(id)
   tagList(
-    h2("Demo of scrollytelling using iris dataset"),
+    h1_tabstop("Demo of scrollytelling using iris dataset"),
```

Comment:
Use accessibility friendly version, also make `h1` as there was no `h1` on the page currently which is an accessibility issue also.